tools/downloader/pytorch_to_onnx.py

```diff
@@ -9,19 +10,28 @@
 import torch.onnx
 
 
+def is_sequence(element):
+    return isinstance(element, (list, tuple))
+
+
 def positive_int_arg(values):
     """Check positive integer type for input argument"""
-    result = []
-    for value in values.split(','):
-        try:
-            ivalue = int(value)
-            if ivalue < 0:
-                raise argparse.ArgumentTypeError('Argument must be a positive integer')
-            result.append(ivalue)
-        except Exception as exc:
-            print(exc)
-            sys.exit('Invalid value for input argument: {!r}, a positive integer is expected'.format(value))
-    return result
+    inputs = ast.literal_eval(values)
+    if not is_sequence(inputs):
+        sys.exit('Model inputs must be a sequence')
```

Comment:
```suggestion
        raise argparse.ArgumentTypeError('{!r}: must be a sequence'.format(inputs))
```

You're supposed to raise an `ArgumentTypeError` if the argument is invalid, not exit immediately. (Yes, I know this function used `sys.exit` before, but since you're revamping it, you might as well fix it.)